_faq/installation_and_deployment/update-cron.md

```diff
@@ -0,0 +1,13 @@
+---
+title: How to update the Cron Job?
```

Comment:
Please lower case `Cron Job`, so it's `cron job`.

Reply:
Done